packages/terser/test/test.js

```diff
@@ -356,3 +358,21 @@ test.serial('terser preserve vars in nameCache when provided', async (t) => {
     }
   });
 });
+
+test.serial('minify when os.cpus().length === 0', async (t) => {
+  const original = os.cpus;
+  os.cpus = () => [];
```

Comment:
Maybe overwriting `os.cpus` in this way is breaking Node 20.

Reply:
@easrng could you please take a look?

Reply:
@easrng should this also include `os.availableParallelism = null;`?